docs/protocol/specifications/vm1.md

```diff
@@ -116,6 +117,8 @@ Execution of any of the following instructions results in immediate failure:
 
 **Run Limit** is a built-in 64-bit integer specifying remaining total cost of execution. Run limit is decreased by the cost of each instruction and also affected by data added to and removed from the data stack and alt stack. Every byte added to either stack costs 1 unit, and every byte removed from either stack refunds 1 unit. (This includes explicit additions and removals by stack-manipulating instructions such as PUSHDATA and DROP, and also implicit additions and removals as when other instructions consume arguments and produce results.)
 
+**Expansion Flag** indicates whether the [expansion opcodes](#expansion-opcodes) are allowed in the program or not. If the flag is off, these opcodes immediately fail the program execution.
```

Comment:
Added clarification here: https://github.com/chain/chain/pull/728/commits/9158fdcbead679db64b478a54d731904a091eb97#diff-a108b3a30ca8a7fff9c31042faeb0761R250

We _will_ have unexecuted instructions due to random jumps. And that's fine.